nodes/Attribute.py

```diff
@@ -1 +1,20 @@
+"""
+Attribute astroid node
+
+To give attribute access.
+
+Attributes:
+    - value  (Name|Other)
+        - Typically a Name node.
+    - attr   (Str)
```

Comment:
The `str` type shouldn't be capitalized (its name is lowercase `str` in Python).